channels-rke2.yaml

```diff
@@ -1825,3 +1877,29 @@ releases:
     charts: &charts-v1-28-7-rke2r1
       <<: *charts-v1-27-11-rke2r1
     featureVersions: *featureVersions-v1
+  - version: v1.28.8-rc1+rke2r1
```

Comment:
Suggesting to replace `v1.28.7+rke2r1` with it instead of adding new as we've not released any v1.28 yet.

Reply:
Replaced.